src/DiffEqSensitivity.jl

```diff
@@ -2,7 +2,7 @@ __precompile__()
 
 module DiffEqSensitivity
 
-using DiffEqBase, Compat, ForwardDiff, ReverseDiff, DiffEqDiffTools, Statistics
+using DiffEqBase, Compat, ForwardDiff, ReverseDiff, Flux, DiffEqDiffTools, Statistics
```

Comment:
Is ReverseDiff needed anymore with this PR?

Reply:
I haven't excised it fully yet, since I don't have test cases for all the places where it's used. But we can work towards that.

Reply:
It was only used for `vecjacobian!`, i.e. for `autovecjac` via seeding. If we're doing this with Flux now, there shouldn't be any more ReverseDiff other than in the tests.

Reply:
No, wait, I wasn't going crazy: https://github.com/JuliaDiffEq/DiffEqSensitivity.jl/blob/f7e5cd179d9b4e9166ab5051ad1f2004500bdc77/src/adjoint_sensitivity.jl#L113

Reply:
why not replace that one with a flux version as well?

Reply:
Sure, I have no issue doing that, it just means there's more than simply deleting code left to be done.

Reply:
This shouldn't be very different from the other 2?